NvimView/NvimView/ModeInfo.swift

```diff
@@ -0,0 +1,50 @@
+/**
+ * Johann Rudloff - @cypheon
+ * See LICENSE
+ */
+
+import MessagePack
+
+public enum CursorShape : Equatable {
+  case Block
```

Comment:
For the sake of consistency, we should use lowercase for the `enum` cases.

Reply:
Done.